src/pykeen/datasets/inductive/ilpc2022.py

```diff
@@ -2,16 +2,29 @@
 
 from docdata import parse_docdata
 
-from .base import DisjointInductivePathDataset
+from .base import UnpackedRemoteDisjointInductiveDataset
 
 __all__ = [
     "ILPC2022Small",
     "ILPC2022Large",
 ]
 
+# ZENODO_URL = "https://zenodo.org/record/6321299/files/pykeen/ilpc2022-v1.0.zip"
```

Comment:
I was not sure whether to use the Zenodo link since it will anyways download both datasets whereas a user might need only one

Reply:
Fine the way it is for now, especially since we maintain the data :)